src/GraphQL/Execution/ExecutionContext.cs

```diff
@@ -38,5 +38,7 @@ public class ExecutionContext : IExecutionContext
         public int? MaxParallelExecutionCount { get; set; }
 
         public Dictionary<string, object> Extensions { get; set; } = new Dictionary<string, object>();
+
+        public IServiceProvider RequestServices { get; set; }
```

Comment:
```suggestion
        private IServiceProvider _requestServices;
        public IServiceProvider RequestServices
        {
            get => _requestServices ?? throw new InvalidOperationException("No scoped service provider specified.  Please set the value of ExecutionOptions.RequestServices to a valid scoped service provider.");
            set => _requestServices = value;
        }
```

Reply:
In the example above, the error message says "scoped service provider", which of course is not strictly necessary.  The error message could read something like this:

> No service provider specified.  Please set the value of the ExecutionOptions.RequestServices to a valid service provider.  Typically, this would be a scoped service provider from your dependency injection framework.

It could even explicitly mention Asp.Net...

Reply:
OK.

Reply:
```suggestion
        private IServiceProvider _requestServices;
        public IServiceProvider RequestServices
        {
            get => _requestServices ?? throw new InvalidOperationException("No service provider specified.  Please set the value of the ExecutionOptions.RequestServices to a valid service provider.  Typically, this would be a scoped service provider from your dependency injection framework.");
            set => _requestServices = value;
        }
```